$NetBSD: patch-lib_CMakeLists.txt,v 1.1 2020/12/14 19:04:41 tnn Exp $

Use devel/cpu_features from pkgsrc since the volk release tarball doesn't
bundle it and we need to patch cpu_features anyway.

--- lib/CMakeLists.txt.orig	2020-11-22 16:33:01.000000000 +0000
+++ lib/CMakeLists.txt
@@ -511,7 +511,7 @@ target_include_directories(volk_obj
     PRIVATE $<BUILD_INTERFACE:${PROJECT_BINARY_DIR}/include>
     PRIVATE $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
     PRIVATE $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/kernels>
-    PRIVATE $<TARGET_PROPERTY:cpu_features,INTERFACE_INCLUDE_DIRECTORIES>
+    PRIVATE $<TARGET_PROPERTY:CpuFeatures::cpu_features,INTERFACE_INCLUDE_DIRECTORIES>
     PRIVATE ${CMAKE_CURRENT_BINARY_DIR}
     PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}
 )
